### DISCUSSION TITLE
Glb model showing ontop of the phone screen

### DISCUSSION THREAD

**Discussion by Kingswillwariboko** [google/model-viewer#4244] [Q&A]
I have a problem where a glb of 1.9mb is displaying too big for in the AR experience
And also some other are freezing the experience

**Comment by elalish** [comment]
glTF specifies that units are meters, so your file was likely modeled in the wrong units and not corrected on export. If you can't fix the source files, use https://modelviewer.dev/docs/index.html#entrydocs-scenegraph-attributes-scale
